### PR TITLE
Better readme file, following aws-api design/steps

### PR DESCRIPTION
**Pull request by souenzzo**

### CODE REVIEW

File: src/compute/gcp/api.clj [hunk 1]
```diff
@@ -31,45 +30,51 @@
...
   (let [op-descriptor (descriptor/get-op-info (::api-descriptor client) op)]
     (-> op-descriptor
-        (select-keys [:description :parameters :responses]))))
-
+      (select-keys [:description :parameters :responses]))))
 
 (defn ops
   [client]
-  (keys (get-in client [::api-descriptor ::descriptor/op->info])))
+  (::descriptor/op->info (::api-descriptor (client-impl/-get-info client))))
```
Comment (kennyjwilli) [inline src/compute/gcp/api.clj:69]: Why this change?

Comment (souenzzo) [reply src/compute/gcp/api.clj:69]: to match aws-api behavior

ops returns a map-of op-name op-description, not a coll-of op-name.
https://github.com/cognitect-labs/aws-api/blob/master/src/cognitect/aws/client/api.clj#L163

Comment (souenzzo) [reply src/compute/gcp/api.clj:69]: Then it will be used in `doc` function. By doing this, our `doc` implementation looks almost identical aws-api.
https://github.com/cognitect-labs/aws-api/blob/master/src/cognitect/aws/client/api.clj#L208
